### PR TITLE
docs: add redirect to our CLA

### PR DESCRIPTION
**Pull request by ghuntley**
## Description

Adds a shortcut redirect to a repository is currently private but will be made public shortly.

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2270"></a>

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /blog/gitlab-integration/+ /blog/gitlab-integration 301
 /careers https://gitpod.crew.work/jobs 301
 /chat https://discord.com/invite/gitpod 301
+/cla https://github.com/gitpod-io/cla 301
```
Comment (Siddhant-K-code) [inline _redirects:4]: [gitpod-io/cla](https://github.com/gitpod-io/cla) is private as of now, it will leads to error 404 page of GH

**Comment by mikenikles** [comment]
@ghuntley What's the status of this?

**Comment by ghuntley** [comment]
Waiting on CLA  process to ship. Any updates here @Siddhant-K-code?

**Comment by Siddhant-K-code** [comment]
maybe, @meysholdt will have some context to it.

**Pull request closed without merging**